<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>864</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1520</width>
    <height>864</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1520</width>
    <height>864</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>7</y>
      <width>1321</width>
      <height>841</height>
     </rect>
    </property>
    <property name="renderHints">
     <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
    </property>
    <property name="viewportUpdateMode">
     <enum>QGraphicsView::MinimalViewportUpdate</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>310</y>
      <width>151</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalButtons">
     <item>
      <layout class="QVBoxLayout" name="addButtons">
       <item>
        <widget class="QPushButton" name="addJunctionButton">
         <property name="text">
          <string>Add Junction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPavementButton">
         <property name="text">
          <string>Add Pavement</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addRoadButton">
         <property name="text">
          <string>Add Road</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addDriverButton">
         <property name="text">
          <string>Add Driver</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPedestrianButton">
         <property name="text">
          <string>Add Pedestrian</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="generateMapButton">
       <property name="text">
        <string>Generate basic map</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="refreshTimeoutLabel">
         <property name="text">
          <string>Refresh timeout</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="refreshTimeoutSpinBox">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>250</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="startStopButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset simulation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>170</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="selectDestinationLayout">
     <item>
      <widget class="QLabel" name="selectDestinationLabel">
       <property name="text">
        <string>Select junction for all drivers and pedestrians to go to</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="selectDestinationComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectDestinationPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Set destination</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
